Dockerfile

```diff
@@ -1,14 +1,19 @@
 FROM nginx:1.10
 MAINTAINER Wyatt Pearsall<Wyatt.Pearsall@cfpb.gov>
+ARG SKIP_JS_BUILD
 
-RUN mkdir -p /usr/src/app
+RUN if [ -z ${SKIP_JS_BUILD+x} ]; then echo "Installing JS deps" && apt-get update && \
+    apt-get install -y curl && \
+    curl -sL https://deb.nodesource.com/setup_7.x | bash - && \
+    apt-get install -y nodejs; fi && mkdir -p /usr/src/app
 
 WORKDIR /usr/src/app
-# COPY . /usr/src/app
+
+COPY . /usr/src/app
 COPY nginx/nginx.conf /etc/nginx/nginx.conf
 
-# RUN chown -R root /usr/src/app && chmod u+rwx /usr/src/app
+RUN if [ -z ${SKIP_JS_BUILD+x} ]; then echo "Building JS" && npm install; fi
```

Comment:
Good catch, npm was failing do to preferring to not run as root. This was actually quite a nice time to switch to running yarn in the dockerfile, so I switched it over (installs in most cases should proceed from the local copies of modules).

You can safely ignore the changes to the various yarn files.